### ISSUE TITLE
[JENKINS-65923] Accessibility issue - Images lacking "alt" text

### ISSUE THREAD

**Issue by timja-bot** [timja-org/jenkins-gh-issues-poc-11-04#1940]
<p>Recent UI upgrades caused the build status icons to lose their useful alt text.</p>

<p>Usability for blind people declined therefore.</p>

<p> </p>

<p>See <a href="https://issues.jenkins.io/browse/JENKINS-65850?focusedCommentId=410509&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-410509" class="external-link" rel="nofollow">https://issues.jenkins.io/browse/JENKINS-65850?focusedCommentId=410509&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-410509</a></p>

---
<details><summary><i>Originally reported by <a title="Alexander Mai" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=maial">maial</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-65923" target="_blank">Accessibility issue - Images lacking "alt" text</a></i></summary>
<i><ul>
<li><b>assignee</b>: <a title="Benny Cheng" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=benebsiny">benebsiny</a>
<li><b>status</b>: Closed
<li><b>priority</b>: Minor
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2021-08-03T17:38:17+00:00
<li><b>imported</b>: 2025-11-04
</ul></i>
</details>

**Comment by timja-bot** [comment]
<i>[Originally related to: <a href="https://github.com/timja-org/jenkins-gh-issues-poc-11-04/issues?q=in%3Atitle%20JENKINS-65850">JENKINS-65850</a>]</i>
